### PR TITLE
feat: upon clicking Save, redirect to Exercises page

### LINKED ISSUE #161: Upon clicking Save, redirect to Exercises page
### Feature Description

When editing an exercise and I hit on the Save button.
image

Redirect to the Exercises page
image

### Use Case

_No response_

### Proposed Solution

_No response_

### Alternatives (optional)

_No response_

### Additional Context

_No response_

### Please check the boxes that apply to this feature request.

- [ ] I have searched the existing feature requests to make sure this is not a duplicate.
- [ ] I have provided a detailed description of the requested feature.
- [ ] I have explained the use case or scenario for this feature.
- [ ] I have included any relevant technical details or design suggestions.
- [ ] I understand that this is a suggestion and that there is no guarantee of implementation.

### PR DESCRIPTION
**Pull request by Toyin5**
## What does this PR do?

fix: Upon clicking Save, redirect to Exercises page

Fixes #161 

## Type of change

- [] Bug fix (non-breaking change which fixes an issue)
- [] Chore (refactoring code, technical debt, workflow improvements)
- [x] Enhancement (small improvements)
- [x] New feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [] This change adds a new database migration
- [] This change requires a documentation update

### Required

- [] Filled out the "How to test" section in this PR
- [x] Self-reviewed my own code
- [x] Commented on my code in hard-to-understand bits
- [x] Ran `pnpm build`
- [x] Checked for warnings, there are none
- [x] Removed all `console.logs`
- [x] Merged the latest changes from main onto my branch with `git pull origin main`
- [x] My changes don't cause any responsiveness issues

### Appreciated

- [] If a UI change was made: Added a screen recording or screenshots to this PR
- [] Updated the ClassroomIO Docs if changes were necessary

### CODE REVIEW

**Comment by Toyin5** [comment]
@rotimi-best pls review

File: apps/dashboard/src/lib/components/Course/components/Lesson/Exercise/index.svelte [hunk 1]
```diff
@@ -61,6 +62,7 @@
         questions: updatedQuestions
       }));
       snackbar.success('Saved Successfully');
+      goto(path);
```
Comment (rotimi-best) [inline apps/dashboard/src/lib/components/Course/components/Lesson/Exercise/index.svelte]:
```suggestion
      snackbar.success('Saved Successfully');

      // Redirect to exercises page
      goto(path);
```

**Review by rotimi-best — CHANGES_REQUESTED** [review]
This looks good, thanks @Toyin5. Just request a small change

**Review by rotimi-best — APPROVED** [review]
Looks good to me

**Pull request merged**